web/src/app/components/home/home.component.html

```diff
@@ -1,6 +1,54 @@
-<div fxLayout="column" fxLayoutAlign="center center" fxLayoutGap="10vh">
-    <span>Welcome to Tribe Dashboard</span>
-    <button mat-flat-button class="mat-primary" (click)="navigateToLogin()">
-    OnBoard
-  </button>
+<div class="homepage-container" fxLayout="column" fxLayoutGap="10vh">
+  <div class="get-started-container">
+    <div>Your social tribe on a single dashboard</div>
+    <div class="primary-text">Connect your socials in one place</div>
+    <button class="get-started-button" mat-button cdkFocusInitial
+      (click)="navigateToLogin()">Get Started</button><br />
+    <img class="pattern-right" src="../../../assets/images/OvalPattern.svg" />
```

Comment:
`assets` should be available publicly, these can be accessed a few ways, for example `assets:github` but not *relative*